### DISCUSSION TITLE
When does WebView2 evergreen gets updated

### DISCUSSION THREAD

**Discussion by vladimir-s-quantori** [MicrosoftEdge/WebView2Feedback#2730] [Q&A]
Hi!

Our app with WebView2 evergreen is installed on several machines and the versions is not the same on any of them, although installation dates of WebView2 are quite close to each other.
I tried to find answer to this question for quite some time and still have nothing:
When we are using evergreen package for WebView2, when exactly it gets updated - during the first installation, whenever we launch it inside the app or by some schedule?

**Comment by victorthoang** [comment]
@vladimir-s-quantori , Evergreen versions of webview updates as soon as it is released monthly. There is a scheduled task that checks the version of WebView on the device and gets updates. For it to update, you'll need to restart the application if you're currently running it and a new update becomes available.
